terraform/cp4aiops/main.tf

```diff
@@ -16,21 +16,30 @@ resource "null_resource" "mkdir_kubeconfig_dir" {
   }
 }
 
+
 module "create_cluster" {
-  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/main/modules/roks"
-  enable               = local.enable_cluster
-  on_vpc               = var.on_vpc
+  source = "github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//modules/roks"
+  enable = local.enable_cluster
+  on_vpc = var.on_vpc
+
+  // General
   project_name         = var.project_name
-  environment          = var.environment
   owner                = var.owner
+  environment          = var.environment
   resource_group       = var.resource_group
   roks_version         = var.roks_version
-  flavors              = var.flavors
-  workers_count        = var.workers_count
-  datacenter           = var.datacenter
+  entitlement          = var.entitled_registry_key
```

Comment:
This is not the same thing as `var.entitled_registry_key`.  This is whether the cluster is being created to install a cloud pak on. See https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/blob/main/modules/roks/variables.tf:38.  This is defaulted to `cloud_pak` which means roks will cost less since an entitlement was purchased for cloud pak.  So you need to just remove this line.

Reply:
I replaced it with this variable: 
```variable "entitlement" {
  default     = ""
  description = "OCP entitlement: leave blank if OCP, set it to `cloud_pak` if cloud pak entitlement"
}

 resource_group       = var.resource_group
  roks_version         = var.roks_version
  entitlement          = var.entitlement
  force_delete_storage = true
```